### PR TITLE
Fix resource Less method

### PR DESCRIPTION
**Pull request by zen-xu**
fix-bug: #1036

Signed-off-by: ZhengYu, Xu <zen-xu@outlook.com>

### CODE REVIEW

**Comment by william-wang** [comment]
It's just a workaround :) 
Suggest to use Rat (xref: https://golang.org/src/math/big/rat.go ) to improve the accuracy.

@Thor-wl , As far as I know, you have been working on this issue a short time ago.

File: pkg/scheduler/api/resource_info.go [hunk 1]
```diff
@@ -224,39 +224,35 @@ func (r *Resource) Multi(ratio float64) *Resource {
...
+				return false
 			}
+
+			same = same && (rQuant == rrQuant)
 		}
-		return true
 	}
-
-	if rr.ScalarResources == nil {
+	if same {
```
Comment (Thor-wl) [inline pkg/scheduler/api/resource_info.go:252]: How about the following testcase?
`
r = {
cpu: 100,
memory: 0,
scalar_resource:{
   gpu:1
}
}
rr = {
cpu: 100,
memory: 100
}
`

Comment (zen-xu) [reply pkg/scheduler/api/resource_info.go:252]: result will be `false`
Why should care about such small resource?

Comment (zen-xu) [reply pkg/scheduler/api/resource_info.go:252]: I'm sorry, it's a bug, I'll fix it

Comment (Thor-wl) [reply pkg/scheduler/api/resource_info.go:257]: > I'm sorry, it's a bug, I'll fix it

In my opinion, current logic has serious problem for not considering some boundary testcases. Fix on current code is not a good idea.

**Pull request closed without merging**